stable/astro/values.yaml

```diff
@@ -1,35 +1,56 @@
 image:
+  # -- Docker image repo
   repository: quay.io/fairwinds/astro
+  # -- Docker image tag
   tag: "v1.5.3"
+  # -- Docker image pull policy
   pullPolicy: IfNotPresent
+# -- The resources block for the deployment.
 resources:
   limits:
     cpu: 100m
     memory: 128Mi
   requests:
     cpu: 100m
     memory: 128Mi
+# -- Deployment ndoeSelector
 nodeSelector: {}
+# -- Deployment tolerations
 tolerations: []
+# -- Deployment affinity
 affinity: {}
 datadog:
+  # -- Datadog API key
   apiKey: ""
+  # -- Datadog app key
   appKey: ""
 rbac:
+  # -- If true, RBAC resources will be created.
   create: true
 deployment:
-  replicas: 2
   serviceAccount:
+    # -- If true, a service account will be created. If false, you must set `deployment.serviceAccount.name`.
     create: true
-    # name: ExistingServiceAccountName
+    # -- The name of an existing service account to use.
+    name:
+  # -- The number of replicas to use.
+  replicas: 2
 secret:
+  # -- If true, a secret with API credentialss will be created. If false, you must set `secret.name`
   create: true
-  # name: ExistingSecretName
+  # -- The name of an existing secret to mount to the container.
+  name:
```

Comment:
Is it OK to leave this blank? previously it was commented out.

Reply:
This is a weird syntax to me. But apparently that's one of the ways to represent nulls in Yaml. This way it still won't have a value.  This way we can still document it.